static/app/types/views.tsx

```diff
@@ -0,0 +1,6 @@
+export type View = {
```

Comment:
I'd make this name more similar to the backend model `GroupSearchView`, `View` is a bit too generic